Common/src/CConfig.cpp

```diff
@@ -1650,8 +1649,8 @@ void CConfig::SetConfig_Options() {
   addBoolOption("ACTDISK_DOUBLE_SURFACE", ActDisk_DoubleSurface, false);
   /* DESCRIPTION: Only half engine is in the computational grid */
   addBoolOption("ENGINE_HALF_MODEL", Engine_HalfModel, false);
-  /* DESCRIPTION: Actuator disk double surface */
-  addBoolOption("ACTDISK_SU2_DEF", ActDisk_SU2_DEF, false);
+  // /* DESCRIPTION: Actuator disk double surface */
+  // addBoolOption("ACTDISK_SU2_DEF", ActDisk_SU2_DEF, false);
```

Comment:
This changes the behavior of SU2_DEF for actuator disks. From not splitting the surface to splitting it. Are you sure this is correct? I would leave it...

Reply:
This keyword is never used, at least not in the repository? Or is it used elsewhere?